packages/now-next/src/index.ts

```diff
@@ -901,17 +887,28 @@ export const build = async ({
       ...staticDirectoryFiles,
     },
     routes: [
-      ...topRoutes,
+      // Before we handle static files we need to set proper caching headers
+      {
```

Comment:
Are these two new routes identical to `topRoutes` or did something change?

Reply:
These are the `topRoutes` moved back in since we don't need to re-apply them now